README.md

```diff
@@ -11,6 +11,41 @@ This is a development only "require-dev" library:
 composer require --dev spryker-sdk/integrator
 ```
 
-## Documentation
+## How it works
+1. Every time when the integrator is run it uploads recipes from `spryker-sdk/integrator-recipes` and uses the master branch. They are uploaded into this folder: `vendor/spryker-sdk/integrator/data/recipes/`.
+2. It gathers the list of dependencies from `composer.json` and their uploaded versions from `composer.lock` of the project.
+3. The integrator checks manifests (recipes) versions and collect manifests that should be applied.
+4. All gathered manifest are applied to the project code.
 
-See [Documentation](docs/).
+## How to test the specific recipe using suite-nonsplit
```

Comment:
This approach is ok for now but needs to be changed. We should also move Spryker only related documentation into confluence.

I already have an idea and will create a ticket for it.